content/swiftui/concepts/colors/colors.md

```diff
@@ -126,3 +126,37 @@ In the example above, the first `ForEach` loop displays a sequence of [`Circle`]
 This will display:
 
 ![SwiftUI Color](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-color.png)
+
+## Using and Defining Custom Colors in SwiftUI
+
+To define custom colors consistently throughout the app in SwiftUI a good way to do this is by creating a custom color under the Assets Folder in XCode.
+
+To ensure the color is cohesive add a Hex Number under the Color Set Tab within the Assets folder as shown below. First, add a color set, and give it a name. Call the new Color to use it. It can be `lightGreen`, `appleGreen`, or in our `brandPrimary` as it will be the brand’s main color used throughout the app.
```

Comment:
```suggestion
To ensure the color is cohesive, add a Hex Number under the Color Set Tab within the Assets folder, as shown below. First, add a color set and give it a name. Call the new Color to use it. It can be `lightGreen`, `appleGreen`, or in our `brandPrimary` as it will be the brand’s main color used throughout the app.
```